host-bin/mount-chroot

```diff
@@ -60,22 +62,34 @@ addtrap "stty echo 2>/dev/null"
 
 # Function to prompt the user for a passphrase. Sets $passphrase.
 promptNewPassphrase() {
+    NOPASS=''
     echo -n "Choose an encryption passphrase for $NAME: " 1>&2
     [ -t 0 ] && stty -echo
-    while [ -z "$passphrase" ]; do
+    while [ -z "$passphrase" -a -z "$NOPASS" ]; do
         read -r passphrase
-        if [ -z "$passphrase" ]; then
-            echo 1>&2
-            echo -n 'You must specify a passphrase: ' 1>&2
-            continue
-        fi
         echo 1>&2
         echo -n 'Please confirm your passphrase: ' 1>&2
         read -r confirmation
+        echo 1>&2
         if [ ! "$confirmation" = "$passphrase" ]; then
             passphrase=''
-            echo 1>&2
             echo -n 'Passphrases do not match; try again: ' 1>&2
+            continue
+        fi
+        if [ -z "$passphrase" ]; then
+            echo 1>&2
+            echo -n \
+'You did not specify a passphrase.  Your keys will not be encrypted.
+Only do this if you are confident that your keys are stored in a secure location.
+Do you want to continue? [y/N] ' 1>&2
+            read -r response
+            echo 1>&2
+            if [ "${response#[Yy]}" = "$response" ]; then
+                continue
+			else
```

Comment:
Sorry, that was gedit trying to use tabs instead of spaces.
Now that I mention it, should I be using something more advanced than gedit to do this type of work?